src/components/reusable/textArea/textArea.ts

```diff
@@ -77,7 +81,14 @@ ${this.value}</textarea
           >
 
           ${this._isInvalid
-            ? html` <kd-icon class="error-icon" .icon=${errorIcon}></kd-icon> `
+            ? html`
+                <kd-icon
+                  class="error-icon"
+                  role="img"
+                  aria-label="Error"
```

Comment:
No hard-coded strings, must be customizable for localization purposes.

Reply:
Addressed